scripts/v1-nxm/v1-nxm-accounting.js

```diff
@@ -0,0 +1,85 @@
+require('dotenv').config();
+const deployments = require('@nexusmutual/deployments');
+const { ethers } = require('hardhat');
+
+const { getContract } = require('./v1-nxm-push-utils');
+
+async function logPoolBalances(provider) {
+  const tokenController = await getContract('TokenController', provider);
+
+  let totalDeposits = ethers.BigNumber.from(0);
+  let totalRewards = ethers.BigNumber.from(0);
+
+  for (let poolId = 1; poolId <= 25; poolId++) {
```

Comment:
we've got new pools in the meantime. please read the number of pools from the factory contract instead of hardcoding.